.github/.codecov.yaml

```diff
@@ -1,13 +1,22 @@
 codecov:
-  require_ci_to_pass: yes
+  require_ci_to_pass: false
   max_report_age: off
 
 coverage:
   precision: 2
-  round: down
+  round: up
   range: "80...100"
   status:
-    patch: off
+    project:
+      default:
+        informational: true
+        target: auto
+        threshold: 10%
```

Comment:
Is 10% threshold too large than needed?

Reply:
I think right now it affects only the color of the report, because either way it is now marked as optional. In my case that I quoted (e.g. Abseil) since I've been adding / touching size'able number of files, the diff was about 7%. I think so is the case for any mass clean-up, like adding headers, or any borrowed library. MAYBE 5% is a better number?

The other thing is default settings quoted by codecov are 80%. Assuming we strive to reach 100%, we may be able to regress down from 100% to -10%, 90% not less than that, with anticipation that a follow-up PR would fix that..

So 90% +/- **10%** should probably be fine. Again, it is informational only. It should only affect the color of the report now.

Reply:
To explain the reason why I saw -7% on my other PR, I re-enabled the metrics, which immediately tanked the score:
https://app.codecov.io/gh/open-telemetry/opentelemetry-cpp/compare/771/tree/api/include/opentelemetry/metrics

Maybe we could indeed deal with something that is closer to 5% instead of 10%.
Please let me know your thoughts on that.